safe_transaction_service/history/models.py

```diff
@@ -1197,6 +1197,9 @@ def without_confirmations(self):
     def trusted(self):
         return self.filter(trusted=True)
 
+    def not_trusted(self):
+        return self.filter(trusted=False)
```

Comment:
should we include executed=False?

Reply:
`executed` transactions should be always `trusted`: https://github.com/safe-global/safe-transaction-service/blob/master/safe_transaction_service/history/indexers/tx_processor.py#L602

Reply:
`executed` transactions should always be  `trusted`: https://github.com/safe-global/safe-transaction-service/blob/master/safe_transaction_service/history/indexers/tx_processor.py#L602